### PR TITLE
ACP2E-994: [Documentation] Add notice that Admin, Bearer and other types of tokens are not supported to skip providing of HTTP X-Captcha header.

### PR DESCRIPTION
**Pull request by vkolesny**
## Purpose of this pull request

This pull request (PR) is to add notice that Admin, Bearer and other types of tokens are not supported to skip providing of HTTP X-Captcha and X-ReCaptcha headers.

According to the clarification:
https://jira.corp.adobe.com/browse/ACP2E-967?focusedCommentId=31270413&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-31270413

Please add following notice:
HTTP X-Captcha and X-ReCaptcha headers:
-Cannot be received by automated script or API call. They are captured by UI based Web API only.
-Can be skipped in protected mutations with integration authorization token only.

## Affected pages

-  https://devdocs.magento.com/guides/v2.4/graphql/protected-mutations.html

## Links to related PRs or Jira tickets

-  ...

## Checklist:

-  [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
-  [ ] My changes follow the code style of this project.
-  [ ] I have read the **CONTRIBUTING** document.
-  [ ] All new and existing tests passed.

### CODE REVIEW

**Review by keharper — APPROVED** [review]

File: src/pages/rest/use-rest/protected-endpoints.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ description: How to supply authentication for protected endpoints that use CAPTC
 
 If CAPTCHA or reCAPTCHA is enabled on pages requiring shopper input, then in most cases, the corresponding endpoints that send requests to the Adobe Commerce server must include an HTTP header that contains a value entered by the shopper (for CAPTCHA) or generated by the Google API (for reCAPTCHA). However, if you specify an integration authorization token in the header of the endpoint, then you do not supply a header specific to CAPTCHA or reCAPTCHA.
 
+HTTP `X-Captcha` and `X-ReCaptcha` headers:
```
Comment (keharper) [inline src/pages/rest/use-rest/protected-endpoints.md:10]:
```suggestion
The HTTP `X-Captcha` and `X-ReCaptcha` headers:
```

File: src/pages/rest/use-rest/protected-endpoints.md [hunk 2]
```diff
@@ -7,6 +7,11 @@ description: How to supply authentication for protected endpoints that use CAPTC
...
+
+*  Cannot be received by automated script or non-UI API call. They are captured and returned by UI Web form only.
```
Comment (keharper) [inline src/pages/rest/use-rest/protected-endpoints.md:12]:
```suggestion
*  Cannot be received by an automated script or a non-UI API call. They are captured and returned by the UI Web form only.
```

File: src/pages/rest/use-rest/protected-endpoints.md [hunk 3]
```diff
@@ -7,6 +7,11 @@ description: How to supply authentication for protected endpoints that use CAPTC
...
+*  Are optional in protected mutation API calls with providing **_integration authorization_** tokens only. They cannot be skipped with Admin and Bearer tokens.
```
Comment (keharper) [inline src/pages/rest/use-rest/protected-endpoints.md:13]:
```suggestion
*  Is optional in protected mutation API calls that provide **_integration authorization_** tokens only. It cannot be skipped when you provide an Admin or Bearer token.
```

**Comment by vkolesny** [comment]
Proposed edits done.

File: src/pages/rest/use-rest/protected-endpoints.md [hunk 4]
```diff
@@ -7,6 +7,11 @@ description: How to supply authentication for protected endpoints that use CAPTC
 
 If CAPTCHA or reCAPTCHA is enabled on pages requiring shopper input, then in most cases, the corresponding endpoints that send requests to the Adobe Commerce server must include an HTTP header that contains a value entered by the shopper (for CAPTCHA) or generated by the Google API (for reCAPTCHA). However, if you specify an integration authorization token in the header of the endpoint, then you do not supply a header specific to CAPTCHA or reCAPTCHA.
 
+The HTTP `X-Captcha` and `X-ReCaptcha` headers:
+
+*  Cannot be received by an automated script or a non-UI API call. They are captured and returned by the UI Web form only.
+*  Is optional in protected mutation API calls that provide **_integration authorization_** tokens only. It cannot be skipped when you provide an Admin or Bearer token.
```
Comment (keharper) [inline src/pages/rest/use-rest/protected-endpoints.md:13]:
```suggestion
*  Are optional in protected mutation API calls that provide **_integration authorization_** tokens only. They cannot be skipped when you provide an Admin or Bearer token.
```

Comment (vkolesny) [reply src/pages/rest/use-rest/protected-endpoints.md]: Fixed.

**Comment by vkolesny** [comment]
Hi @keharper ,
All changes committed.
Ready for review.

**Review by keharper — APPROVED** [review]

**Comment by keharper** [comment]
running tests

**Pull request merged**